examples/jenkins/pipeline/nodejs-sample-pipeline.yaml

```diff
@@ -0,0 +1,78 @@
+kind: "BuildConfig"
+apiVersion: "v1"
+metadata:
+  name: "sample-nodejs-pipeline"
```

Comment:
align this name and the filename.

Reply:
updated, thought i had already fixed that, must have been in the docs